javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -2633,8 +2647,27 @@ public static ExpressionTree yieldTreeGetValue(Tree yieldTree) {
      * @return true if the variableTree is declared using the {@code var} Java keyword
      */
     public static boolean isVariableTreeDeclaredUsingVar(VariableTree variableTree) {
+        // The JCVariableDecl class has the method "declaredUsingVar()" returns true if the
+        // variable was declared using the "var" keyword.
+        // The start positions are set for types of variables declared using var in
+        // https://github.com/openjdk/jdk/commit/e2f736658fbd03d2dc2186dbd9ba9b13b1f1a8ac
+        // Checking the pos is equal to Position.NOPOS will return the wrong result after above
+        // Javac change.
         JCExpression type = (JCExpression) variableTree.getType();
-        return type != null && type.pos == Position.NOPOS;
+        if (type == null) {
+            return false;
+        } else if (JCVARDECL_DECLAREDUSINGVAR != null && !isErroneousTree(type)) {
```

Comment:
Please create an issue that describes the problem with erroneous trees and add a reference to that issue here. When we fix that issue, we should remove this hack here.